### ISSUE TITLE
The documentation on how to run this is lacking

### ISSUE THREAD

**Issue by 0xR** [adamtornhill/ownership-fractals#1]
I want to try to run this on a code base. I have tried to read the sourcecode on how to run it, but it has been challenging. After some trial and error I figured out I should run code-maat like this to get my input data:

```
code-maat -c git2 -l myrepo.log -a entity-effort > entity-effor.csv
```

Now I don't know how to run `ownership-fractal` or how to pass it the data. It looks like I need 2 another input file with the author colors. But I have no clue on how to pass it to the program. There doesn't seem to be any CLI options.

**Comment by adamtornhill** [comment]
You're right - there's no CLI to this program. This code wasn't meant for public consumption. It's a prototype I did to play around with quil.

If you want to run the code you need to start a Clojure REPL and call the `visualize` function from that REPL.

I also did a [Processing prototype](https://github.com/adamtornhill/FractalFigures) that prompts you for the input files needed for the fractals. Another alternative is to use the [CodeScene service](https://codescene.io/) which automates the generation of fractal figures.
